### PR TITLE
How to contribute

### PR DESCRIPTION
**Pull request by Jinjiang**
Downgraded `/about` page level and updated `README.md`

### CODE REVIEW

**Comment by Jinjiang** [comment]
@skyline0705 @Justineo for you

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # vue.js 中文文档
```
Comment (Justineo) [inline README.md]: `vue.js` → `Vue.js`

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
-https://cn.vuejs.org
+该站点基于 [hexo](https://hexo.io/) 构建而成。网站内容在 `src` 文件夹内，格式为 Markdown。欢迎 issue 或 pull request。
```
Comment (Justineo) [inline README.md]: `hexo` → `Hexo`

**Comment by Jinjiang** [comment]
Done

**Pull request merged**